front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -202,11 +202,45 @@ public Collection<Pipeline> getTriggeredPipelines(
   public List<Pipeline> listByApplication(
       @PathVariable(value = "application") String application,
       @RequestParam(value = "pipelineNameFilter", required = false) String pipelineNameFilter,
-      @RequestParam(required = false, value = "refresh", defaultValue = "true") boolean refresh) {
+      @RequestParam(required = false, value = "refresh", defaultValue = "true") boolean refresh,
+      @RequestParam(required = false, value = "pipelineStateFilter", defaultValue = "all")
```

Comment:
What are the other possible values for pipelineStateFilter?  Unless there's something super obvious coming down the road soon, I'd rather use a `boolean`.  Plus also, even though `refresh` does it, I don't think we need/want to specify value if it's the same as the argument name.

Reply:
It is `all`,`enabled`,`disabled`. The reason i switched from boolean is that i can see a useful predefined filter in Deck -> Gate for getting only the enabled or the disabled pipelines. 

But we can always leave this as a future improvement if it is preferred to keep it cleaner for now.

Reply:
Even if this info comes from gate (which would mean a change to https://github.com/spinnaker/orca/pull/4819 right?), if those are the values, then a Boolean is enough, isn't it?

Reply:
Gate directly calls Front50 for the pipelineConfigIds and then calls Orca to fetch the executions for those pipelineConfigIds.So both changes are needed.

Yes,sure i can change to Boolean with null fetching everything and true|false for the enabled|disabled

Reply:
Pushed the Boolean change here and in the orca PR